fadin/utils/compute_constants.py

```diff
@@ -60,14 +60,14 @@ def _get_ztzG(events_grid, L):
             ej = events_grid[j]
             for tau in range(L):
                 for tau_p in range(tau + 1):
-                    if tau_p == 0:
-                        if tau == 0:
-                            ztzG[i, j, tau, tau_p] = ei @ ej
-                        else:
-                            ztzG[i, j, tau, tau_p] = ei[:-tau] @ ej[tau:]
-                    else:
-                        diff = tau - tau_p
-                        ztzG[i, j, tau, tau_p] = ei[:-tau] @ ej[diff:-tau_p]
+                    # if tau_p == 0:
+                    #     if tau == 0:
+                    #         ztzG[i, j, tau, tau_p] = ei @ ej
+                    #     else:
+                    #         ztzG[i, j, tau, tau_p] = ei[:-tau] @ ej[tau:]
+                    # else:
```

Comment:
don't commit commented blocks like this. Either remove them or add a comment for why you keep them.